python/bindings/src/spark_dsg_bindings.cpp

```diff
@@ -184,6 +184,8 @@ PYBIND11_MODULE(_dsg_bindings, module) {
       .def("__repr__", [](const NodeSymbol& ns) { return ns.str(false); })
       .def("__hash__",
            [](const NodeSymbol& symbol) { return static_cast<NodeId>(symbol); })
+      .def("str",
+           [](const NodeSymbol& ns, const bool literal) { return ns.str(literal); })
```

Comment:
You can probably bind the str function directly, i.e., 
```suggestion
      .def("str", &NodeSymbol::str, "literal"_a = true);
```